lib/link_thumbnailer/page.rb

```diff
@@ -18,6 +18,7 @@ def initialize(url, options = {})
 
     def generate
       @source = processor.call(url)
+      processor.http.shutdown
```

Comment:
should probably be done in processor at the end of the `call` action.

Reply:
I can't do it here. If the website has redirection call calls twice. If we need to place it in the processor, I can do it only in the selected area
![image](https://user-images.githubusercontent.com/20104771/53413015-99a32780-39d3-11e9-93fb-7e884151d113.png).
Is it ok? Or it will be better to leave it like now.

Reply:
Ah i see, i would create a new method in processor then that the `generate` method would call and use `call` for the recursive loop:

```
# page.rb
def generate
  @source = processor.call(url)
end

# processor.rb
def start(url)
  call(url)
  shutdown
end
def shutdown
  http.shutdown
end
def call(...)
  # same as now
end
```

Reply:
Ok. Thanks.

Reply:
Already fix. Thanks for the advice.